data/menus.yaml

```
@@ -1,15 +1,24 @@
+- title: Community
+  items:
+    - Zulip chat: https://leanprover.zulipchat.com/
+    - GitHub: https://github.com/leanprover-community/mathlib
+    - Community information: meet.html
+    - Papers about Lean: papers.html
+    - Projects using Lean: lean_projects.html
+
 - title: Installation
   items:
     - Get started: get_started.html
-    - Linux installation: install/linux.html
-    - Debian installation: install/debian.html
+    - Debian/Ubuntu installation: install/debian.html
+    - Generic Linux installation: install/linux.html
     - MacOS installation: install/macos.html
     - Windows installation: install/windows.html
     - Online version (no installation): https://leanprover-community.github.io/lean-web-editor
 
 - title: Documentation
   items:
-    - Big list of docs (start here): learn.html
+    - Leaning resources (start here): learn.html
```

```suggestion
    - Learning resources (start here): learn.html
```

Unless this is supposed to be a clever pun.